frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx

```diff
@@ -993,22 +993,14 @@ const ManageHostsPage = ({
     let action = hostsAPI.transferToTeam(teamId, selectedHostIds);
 
     if (isAllMatchingHostsSelected) {
-      let status = "";
-      let labelId = null;
-      const selectedStatus = getStatusSelected();
+      const labelId = selectedLabel?.id as number;
```

Comment:
Is there any way to remove this `as` and handle the types? I'm guessing without it `lableId` can be `number` or `undefined`. I think we should handle the types instead of type casting so something like

```js
const labelId = selectedLabel?.id as number;
if (typeof labelId === "undefined") {
 //  ...do something 
}

action = hostsAPI.transferToTeamByFilter({
// ...
```

I'd say we want to avoid type casting unless we are 100% sure of what the type will be.

Reply:
Cleaned up typing here to type number or undefined since string is being deprecated when status is no longer considered a "label"